src/app/DTT/upload/Dropzone.js

```diff
@@ -46,48 +46,42 @@ const DropZone = ({ handleSave, closeDropzone, maxSize = '5000000', maxFiles = 1
 
     return (
       <Flex key={name} p="4" background="teal.50">
```

Comment:
Remove this background color. As we have white color text on dark mode, user won't be able to see the text in this background.